nss-day-cohort-46/rare-rest-rare-api-cool-cake
# Pp post reactions

Details:
Backend for adding reactions to posts is complete.

Testing Instructions:
Postman URL:
```
http://localhost:8000/posts/1/react
```

Postman Object:
```
{
    "reactionId": 4
}
```

Make a POST request and see if the reaction is added to the PostReaction table in the database.

Then, change the request to DELETE and see if the reaction was removed from the table.
